bigsea-scaler/service/api/actuator/plugins/remote_KVM_tunnel.py

```diff
@@ -35,4 +54,18 @@ def get_allocated_resources(self, host_ip, vm_id):
         except:
             # FIXME: review this exception type
             raise Exception("Could not get allocated resources")
-        
\ No newline at end of file
+            
+    def get_io_quota(self, host_ip, vm_id):
+        command = "virsh blkdeviotune %s" % (vm_id)
+        command += " \"`virsh domblklist %s | awk 'FNR == 3 {print $1}'`\"" % (vm_id)
```

Comment:
There are some wrong characters in my review comment because of the limitations of markdown here in github. So, try to follow the idea to use only one statement and broke the string for each command jumping lines using "+" in the end of each line.